### ISSUE TITLE
Document reply_to

### ISSUE THREAD

**Issue by Gerardwx** [ecederstrand/exchangelib#944]
Setting a _reply_to_ in a Message turned out to be quite simple, but I didn't find it documented anywhere -- I ended up just guessing 
`   m = Message(account=account,
                    folder = account.sent,
                    subject="Test mail r",
                    body="Test email {}".format(datetime.datetime.now()),
                    reply_to=['support@example.com'],
                    to_recipients=[address],
                    bcc_recipients=['hidden@example.com']
                    )`
                   
would work and was happy it did. Might save the next fellow a little time if it was documented explicitly.

**Comment by ecederstrand** [comment]
How would you suggest that it's documented? I'm hesitant to add documentation for all fields since exchangelib is a client for EWS and the fields are pretty well described in the EWS documentation.

**Comment by Gerardwx** [comment]
If I understand what you're saying correctly, exchangelib is a (very convenient) wrapper around the EWS Soap protocol? Then a note somewhere in the main documentation referring users to the Microsoft Docs would be helpful ( e.g. https://docs.microsoft.com/en-us/exchange/client-developer/web-service-reference/ews-reference-for-exchange)

**Comment by ecederstrand** [comment]
All classes in exchangelib already have a link to MSDN docs in their docstring, e.g. https://ecederstrand.github.io/exchangelib/exchangelib/items/message.html#exchangelib.items.message.Message Is that what you're requesting?

**Comment by ecederstrand** [comment]
Closing, but feel free to reopen if you have suggestions for better documentation of fields in general.

**Comment by Gerardwx** [comment]
Erik,
That's fine. This a great library (just works) and I appreciate the effort.

I think the reason I didn't find what I was looking for is when I started at https://ecederstrand.github.io/exchangelib/, I didn't follow the link into source code documentation. In most projects I've used the source code documentation is oriented towards users who wish to extend the library functionality as opposed to using existing features.
